crates/red_knot_python_semantic/src/types.rs

```diff
@@ -233,31 +233,39 @@ fn declarations_ty<'db>(
 /// Unique ID for a type.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
 pub enum Type<'db> {
-    /// the dynamic type: a statically-unknown set of values
+    /// The dynamic type: a statically-unknown set of values
     Any,
-    /// the empty set of values
+    /// The empty set of values
     Never,
-    /// unknown type (either no annotation, or some kind of type error)
-    /// equivalent to Any, or possibly to object in strict mode
+    /// Unknown type (either no annotation, or some kind of type error)
+    /// Equivalent to Any, or possibly to object in strict mode
     Unknown,
-    /// name does not exist or is not bound to any value (this represents an error, but with some
-    /// leniency options it could be silently resolved to Unknown in some cases)
+    /// Name does not exist or is not bound to any value (this represents an error, but with some
+    /// Leniency options it could be silently resolved to Unknown in some cases)
```

Comment:
this change isn't correct: this line is part of the same sentence as the previous line

```suggestion
    /// leniency options it could be silently resolved to Unknown in some cases)
```